cmd/autoheal/active_jobs_worker.go

```diff
@@ -19,23 +19,30 @@ package main
 import (
 	"github.com/golang/glog"
 	"k8s.io/apimachinery/pkg/util/runtime"
+
+	autoheal "github.com/openshift/autoheal/pkg/apis/autoheal/v1alpha2"
```

Comment:
I have just merged #69, which means that you should use the `.../apis/autoheal` package, without the version number. Please change that.

Reply:
Thanks, fixed.